tests/e2e/ingress/00-install.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   mode: "deployment"
   ingress:
     type: ingress
-    hostname: "example.com"
+    ruleType: subdomain
```

Comment:
can we make this a different ingress test to verify no breaking changes for users?

Reply:
done I have created a new test for subdomains